Source/WebKit/Platform/IPC/StreamConnectionWorkQueue.h

```diff
@@ -69,6 +69,7 @@ class WTF_CAPABILITY("is current") StreamConnectionWorkQueue final : public Func
     mutable Lock m_lock;
     RefPtr<Thread> m_processingThread WTF_GUARDED_BY_LOCK(m_lock);
     Deque<Function<void()>> m_functions WTF_GUARDED_BY_LOCK(m_lock);
+    std::optional<WTF::Function<void()>> m_cleanup WTF_GUARDED_BY_LOCK(m_lock);
```

Comment:
Should just be `WTF::Function<void()>`, not need to wrap in an optional. a Function has a null state and can be null-checked.

Reply:
Can we call this m_cleanupFunction or something similar? Just m_cleanup doesn't seem great.